modules/system/models/maillayout/fields.yaml

```diff
@@ -39,3 +39,8 @@ secondaryTabs:
             size: giant
             tab: system::lang.mail_templates.content_text
             stretch: true
+
+        inline_css:
```

Comment:
you can store this as `options[inline_css]` and then access it with `@$layout->options['inline_css'];` which removes the need for the getter and setter methods.